src/local-notifications.android.ts

```diff
@@ -215,6 +215,10 @@ export class LocalNotificationsImpl extends LocalNotificationsCommon implements
             options.color = options.color.android;
           }
 
+          if (options.notificationLed && options.notificationLed.ledColor){
+            options.notificationLed.ledColor = options.notificationLed.ledColor.android;
```

Comment:
Please see [this discussion](https://github.com/EddyVerbruggen/nativescript-local-notifications/pull/133/files#r249005541).
I Am trying to follow [this](https://github.com/EddyVerbruggen/nativescript-local-notifications/pull/133/files#diff-75678843cd805744830e5ddc817a4a44R215)